common/scripts/docker/startFakeAPI.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash -e
+
+source "$CONFIG_DIR/admiral.env"
```

Comment:
why do we need to source this? if any envs are required, they should be passed from the route (like CONFIG_DIR, SCRIPTS_DIR etc)